deploy/gateway/values.schema.json

```diff
@@ -2,7 +2,7 @@
   "$schema": "https://json-schema.org/schema#",
   "type": "object",
   "title": "JSON schema for Twingate Gateway Helm Chart values",
-  "required": ["twingate", "tls"],
```

Comment:
need to also add defaults in template?

Reply:
Double-checked that this change was not needed. When the user of the Helm chart doesn't specify `tls` object, the default values would be used. Helm lint doesn't fail either. The error `Schema validation: Missing required property 'tls'` only appears in certain editor...